packages/dds/tree/src/schema-stored/Stored and View Schema.md

```diff
@@ -230,7 +251,9 @@ And with CircleV1, we provide an adapter for use with schematize that can handle
 
```

Comment:
```suggestion

### How to deal with rebasing of schema changes?

Like any other edit, schema changes can be concurrent and need rebasing.
Doing this will need carful design, both to schema violation in documents, but also to provide nice app facing API for handling cases where their schema changes conflicted and thus were apparently undone in an incompatible way.
These cases (where a schema change gets reverted/undone or conflicted) mean that the monotonic approach of schema changes always relax constraints cannot apply everywhere, and thus may be worth reconsidering.

```